packages/components-native/src/InputText/InputText.test.tsx

```diff
@@ -528,3 +529,102 @@ describe("InputText", () => {
     });
   });
 });
+
+describe("Transform", () => {
+  const base64Transformer = {
+    input: (a: string | undefined) => {
+      if (!a) return a;
+
+      return atob(a);
+    },
+    output: (a: string | undefined) => {
+      if (!a) return a;
+
+      return btoa(a);
+    },
+  };
+
+  describe("when working with controlled components", () => {
+    it("form state gets corect value during change", () => {
+      const a11yLabel = "Test InputText";
+      const onFormValueUpdate = jest.fn();
+      const { getByLabelText } = renderInputText({
+        onChangeText: onFormValueUpdate,
+        accessibilityLabel: a11yLabel,
+        transform: base64Transformer,
+      });
+      const input = getByLabelText(a11yLabel);
+      fireEvent.changeText(input, "New value ");
+      expect(onFormValueUpdate).toHaveBeenCalledWith("TmV3IHZhbHVlIA==");
+    });
+
+    it("form state gets corect value on blur", () => {
+      const a11yLabel = "Test InputText";
+      const onFormValueUpdate = jest.fn();
+      const { getByLabelText } = renderInputText({
+        onChangeText: onFormValueUpdate,
+        accessibilityLabel: a11yLabel,
+        transform: base64Transformer,
+      });
+      const input = getByLabelText(a11yLabel);
+      fireEvent.changeText(input, "New value ");
+      fireEvent(input, "blur");
+      expect(onFormValueUpdate).toHaveBeenCalledTimes(2);
+      expect(onFormValueUpdate).toHaveBeenLastCalledWith("TmV3IHZhbHVl");
+    });
+  });
+
+  describe("When working with uncontrolled components", () => {
+    const onFormValueUpdate = jest.fn();
+    beforeEach(jest.clearAllMocks);
+
+    function FormWrapper({ children }: { readonly children: React.ReactNode }) {
+      const form = useForm();
+      useEffect(() => {
+        return form.watch((value, { name, type }) => {
```

Comment:
```suggestion
        return form.watch(value => {
```

Reply:
Ah thanks I just pulled to see why

Reply:
I blame me not paying attention and the linting failure output being obtuse.